index.js

```diff
@@ -1,3 +1,9 @@
 function subtract(a, b) {
   return a - b;
-}
\ No newline at end of file
+}
+
+const soma = ( x, y) => {
+
+
+  return x + y
+}
```

Comment:
Oiiii Edu.. blz? Então vamos add o lint nos combinados para o time todo usar. Faz sentido?
- Um ponto legal seria usarmos sempre functions ao invés de arrow functions.
😄 
```suggestion
function soma(x, y) {
  return x + y;
}
```

Reply:
Obrigado Maite! Irei corrigir agora!